src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -202,7 +204,7 @@ public static class EditPatientDescriptor {
         private Note note;
         private Name nokName;
         private Phone nokPhone;
-        private List<Appt> appts = new ArrayList<>();
```

Comment:
This looks good because we will fix the EditCommand problem of not throwing error for empty fields 👍